app/src/main/java/com/github/se/project/ui/components/LessonLocationDisplay.kt

```diff
@@ -169,8 +169,8 @@ fun calculateDistance(userLocation: LatLng?, lessonLocation: LatLng): Float {
 }
 
 /**
- * Private function to adjust the zoom level based on the distance between two locations. The
- * further the locations, the lower the zoom level.
+ * Private function to adjust evel based on the distance between two locations. The further the
```

Comment:
whoops, typo here I guess 
```suggestion
 * Private function to adjust level based on the distance between two locations. The further the
```